### ISSUE TITLE
Editor not possible?

### ISSUE THREAD

**Issue by sindhus** [youurayy/gfms#30]
hi! thanks for writing gfms. I tried out some online markdown editors like [dillinger](dillinger.io) and [stackedit] (https://stackedit.io/editor). Your module is nice to view markdown files offline but I was wondering, is it not possible to edit them through the web UI too?

**Comment by youurayy** [comment]
No, that functionality was never planned. Also because the offline rendering is not perfect, and 100% results are only achievable with having GithHub API render the markdown for you.
